pkg/util/version.yaml

```diff
@@ -0,0 +1 @@
pi.github.com/repos/flyteorg/flytectl/releases/assets/39503192","id":39503192,"node_id":"MDEyOlJlbGVhc2VBc3NldDM5NTAzMTky","name":"flytectl_Windows_i386.zip","label":"","uploader":{"login":"flyte-bot","id":78108056,"node_id":"MDQ6VXNlcjc4MTA4MDU2","avatar_url":"https://avatars.githubusercontent.com/u/78108056?v=4","gravatar_id":"","url":"https://api.github.com/users/flyte-bot","html_url":"https://github.com/flyte-bot","followers_url":"https://api.github.com/users/flyte-bot/followers","following_url":"https://api.github.com/users/flyte-bot/following{/other_user}","gists_url":"https://api.github.com/users/flyte-bot/gists{/gist_id}","starred_url":"https://api.github.com/users/flyte-bot/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/flyte-bot/subscriptions","organizations_url":"https://api.github.com/users/flyte-bot/orgs","repos_url":"https://api.github.com/users/flyte-bot/repos","events_url":"https://api.github.com/users/flyte-bot/events{/privacy}","received_events_url":"https://api.github.com/users/flyte-bot/received_events","type":"User","site_admin":false},"content_type":"application/zip","state":"uploaded","size":9025968,"download_count":0,"created_at":"2021-06-30T19:17:04Z","updated_at":"2021-06-30T19:17:05Z","browser_download_url":"https://github.com/flyteorg/flytectl/releases/download/v0.1.29/flytectl_Windows_i386.zip"},{"url":"https://api.github.com/repos/flyteorg/flytectl/releases/assets/39503188","id":39503188,"node_id":"MDEyOlJlbGVhc2VBc3NldDM5NTAzMTg4","name":"flytectl_Windows_x86_64.zip","label":"","uploader":{"login":"flyte-bot","id":78108056,"node_id":"MDQ6VXNlcjc4MTA4MDU2","avatar_url":"https://avatars.githubusercontent.com/u/78108056?v=4","gravatar_id":"","url":"https://api.github.com/users/flyte-bot","html_url":"https://github.com/flyte-bot","followers_url":"https://api.github.com/users/flyte-bot/followers","following_url":"https://api.github.com/users/flyte-bot/following{/other_user}","gists_url":"https://api.github.com/users/flyte-bot/gists{/gist_id}","starred_url":"https://api.github.com/users/flyte-bot/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/flyte-bot/subscriptions","organizations_url":"https://api.github.com/users/flyte-bot/orgs","repos_url":"https://api.github.com/users/flyte-bot/repos","events_url":"https://api.github.com/users/flyte-bot/events{/privacy}","received_events_url":"https://api.github.com/users/flyte-bot/received_events","type":"User","site_admin":false},"content_type":"application/zip","state":"uploaded","size":9390673,"download_count":0,"created_at":"2021-06-30T19:17:03Z","updated_at":"2021-06-30T19:17:04Z","browser_download_url":"https://github.com/flyteorg/flytectl/releases/download/v0.1.29/flytectl_Windows_x86_64.zip"}],"tarball_url":"https://api.github.com/repos/flyteorg/flytectl/tarball/v0.1.29","zipball_url":"https://api.github.com/repos/flyteorg/flytectl/zipball/v0.1.29","body":"## Changelog\n\nd6b262a Added exec command in sandbox (#122)\n\n"}
```

Comment:
what's the purpose of this file?

Reply:
Was an accidental checkin. Thanks for pointing it out. Removed it